packages/lib-core/src/utils/k8s/k8s-get-resource.ts

```diff
@@ -0,0 +1,13 @@
+const abbrBlacklist = ['ASS'];
+
+/**
+ * @deprecated - This is SDK internal function for resource-icon purposes, use Console version instead.
+ * Provides an abbreviation string for given kind with respect to abbrBlacklist.
+ * @param kind Kind for which the abbreviation is generated.
+ * @return Abbreviation string for given kind.
+ * TODO: Use in resource-icon component once it is being migrated to the SDK.
+ * * */
+export const kindToAbbr = (kind: string) => {
+  const abbrKind = (kind.replace(/[^A-Z]/g, '') || kind.toUpperCase()).slice(0, 4);
+  return abbrBlacklist.includes(abbrKind) ? abbrKind.slice(0, -1) : abbrKind;
+};
```

Comment:
If it's not needed -- we should be adding it yet.

Delete this for now. We need to think on the `@deprecated` state too. We need this internally, and what is and isn't exposed out of the SDK is on us, so this deprecation is weird for the Core SDK.